code/__HELPERS/piping_colors_lists.dm

```diff
@@ -0,0 +1,31 @@
+///All colors available to pipes and atmos components
+GLOBAL_LIST_INIT(pipe_paint_colors, sortList(list(
+	"amethyst" = COLOR_AMETHYST, //supplymain
```

Comment:
Why does only this one have a comment? What does it mean?

Reply:
Nothing is a remnant of the old code, probably amethyst was used as supply pipe.

Want me to remove it?

Reply:
done, next commit